docs/kyma/docs/04-04-gke-installation.md

```diff
@@ -6,17 +6,46 @@ type: Installation
 This Installation guide shows developers how to quickly deploy Kyma on a [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) (GKE) cluster. Kyma installs on a cluster using a proprietary installer based on a Kubernetes operator.
 
 ## Prerequisites
-- A domain for your GKE cluster
 - [Google Cloud Platform](https://console.cloud.google.com/) (GCP) project
 - [Docker](https://www.docker.com/)
 - [Docker Hub](https://hub.docker.com/) account
 - [gcloud](https://cloud.google.com/sdk/gcloud/)
+- A domain for your GKE cluster (optional)
 
->**NOTE:** If you don't own a domain which you can use or you don't want to assign a domain to a cluster, see [this](#installation-install-kyma-on-a-gke-cluster-with-wildcard-dns) document which shows you how to create a cluster-based playground environment using a wildcard DNS provided by xip.io.
+## Prepare the GKE cluster
+
+1. Select a name for your cluster. Set the cluster name and the name of your GCP project as environment variables. Run:
+    ```
+    export CLUSTER_NAME={CLUSTER_NAME_YOU_WANT}
+    export PROJECT={YOUR_GCP_PROJECT}
+    ```
+
+2. Create a cluster in the `europe-west1` region. Run:
+    ```
+    gcloud container --project "$PROJECT" clusters \
+    create "$CLUSTER_NAME" --zone "europe-west1-b" \
+    --cluster-version "1.11.5" --machine-type "n1-standard-2" \
+    --addons HorizontalPodAutoscaling,HttpLoadBalancing,KubernetesDashboard
+    ```
+
+3. Install Tiller on your GKE cluster. Run:
+
+    ```
+    kubectl apply -f installation/resources/tiller.yaml
+    ```
 
-## DNS setup
+4. Add your account as the cluster administrator:
+    ```
+    kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=$(gcloud config get-value account)
+    ```
+    
+## DNS setup and TLS certificate generation
 
-Delegate the management of your domain to Google Cloud DNS. Follow these steps:
+>**NOTE:** By default, Kyma installs on a cluster using a wildcard DNS provided by `xip.io`. If you don't own a domain which you can use or you don't want to assign your domain to a cluster, skip this section.
```

Comment:
This piece of information should be added to the very beginning of the document. The note here could be a bit shorter and guide to the next section the user should read when using XIP.

Reply:
```suggestion
>**NOTE:** Execute instructions from this section only if you want to use your own domain. Otherwise, proceed to [this](#installation-install-kyma-on-a-gke-cluster-prepare-the-installation-configuration-file) section. 
```